packages/compass-aggregations/src/components/pipeline/collation-toolbar.module.less

```diff
@@ -60,6 +60,11 @@
   input:focus {
     outline-width: 0;
   }
+
+  .max-time-ms {
```

Comment:
Maybe we can [hide the arrow buttons](https://css-tricks.com/snippets/css/turn-off-number-input-spinners/) here to make it more like the one in the query bar